src/components/ContactUsForm/index.css

```diff
@@ -0,0 +1,58 @@
+.contactUsForm {
+  background: #add4c5;
+  display: flex;
+  justify-content: center;
+}
+
+.contactUsForm__content__label {
+  font-family: "Raleway";
+  font-size: clamp(1rem, 1.5vw, 2rem);
+  color: white;
+  margin-top: 0.5rem;
+  text-align: left;
+}
+
+.contactUsForm__content__input,
+.contactUsForm__content__textarea {
+  font-family: "Raleway";
+  font-size: clamp(1.5rem, 2vw, 3rem);
+  box-sizing: border-box;
+  border: 2px solid #ccc;
+  border-radius: 0.5rem;
+  font-size: 16px;
+  width: clamp(20vw, 30vw, 40vw);
+  height: clamp(3vh, 5vh, 6vh);
+}
+.contactUsForm__content__textarea {
+  height: clamp(8vh, 10vh, 16vh);
+}
```

Comment:
Louis warned us about use of viewport

Reply:
Thanks for reminding;) I updated all viewport scales into rem.